### PR TITLE
Align Config and State Management

### PR DESCRIPTION
**Pull request by mohitsoni**

### CODE REVIEW

File: cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java [hunk 1]
```diff
@@ -149,15 +150,7 @@
...
-    public static final List<Map<String, Object>> DEFAULT_SEED_PROVIDER =
-            ImmutableList.<Map<String, Object>>of(
-                    ImmutableMap.<String, Object>of(
-                            "class_name",
-                            "org.apache.cassandra.locator.SimpleSeedProvider",
-                            "parameters", ImmutableList.of(ImmutableMap.of
-                                    ("seeds", "127.0.0.1"))
-                    )
-            );
+    public static final String DEFAULT_SEEDS_URL = "http://cassandra.marathon.mesos:8080/v1/seeds";
```
Comment (mohitsoni) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java]: Can we default this inside `scheduler.yml` ?

Comment (kow3ns) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java]: Don't need it anymore. I'm moving the configuration back closer to the original. The motivations for some previous implementation are no longer relevant.

Comment (mohitsoni) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java]: Okay.

Comment (kow3ns) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java]: Okay I'm merging this to master. I've already pulled this into the 3.0.7 branch and its testing there. I'll rebase off of master.

**Pull request merged**